env/default.json

```diff
@@ -1,2 +1,7 @@
 {
-}
\ No newline at end of file
+}
+
+
+
```

Comment:
@oscarmild 
**why is this empty space?**

Reply:
@fromgopi 
we need to make sure that some of the default items like ports env and other stuff should be placed.

Reply:
@oscarmild Ok
we can common out all the items like logger path, version name etc.

Reply:
@fromgopi Sure I will make sure that all the default items will go into this file.

Reply:
@oscarmild Ok.